packages/user-ui/src/pages/project/page.rs

```diff
@@ -2,14 +2,280 @@ use dioxus::prelude::*;
 use dioxus_logger::tracing;
 use dioxus_translate::Language;
 
-use crate::pages::project::controller;
+use crate::{
+    components::icons::{
+        download::DownloadIcon,
+        pencil::Pencil,
+        person::Person,
+        right_arrow::RightArrow,
+        triangle::{TriangleDown, TriangleUp},
+    },
+    pages::project::controller,
+};
 
 #[component]
 pub fn ProjectPage(lang: Language, project_id: i64) -> Element {
     let ctrl = controller::Controller::init(lang)?;
     let deliberation = ctrl.get_deliberation();
     tracing::debug!("deliberation: {:?}", deliberation);
     rsx! {
-        div { "project page" }
+        div {
+            ProjectProfile { lang }
+            ProjectDetailsMenu {}
+        }
+    }
+}
+
+#[component]
+pub fn ProjectProfile(lang: Language) -> Element {
+    rsx! {
+        div { class: "w-[1300px] h-[300px] mb-[40px] flex flex- row justify-center items-center gap-[40px]",
```

Comment:
width of this component should be inherited from the page component.

And it never be fixed to constant number. It will be broken in mobile.

Instead of it, use `w-full max-w-[1300px]` if the max size is desired to 1300px.